packages/wallet/src/services/TipTracker.ts

```diff
@@ -62,20 +66,34 @@ export class TipTracker extends PersistentDocumentTrackerSubject<Cardano.Tip> {
           connectionStatus$
         ]).pipe(
           // Throttle syncing by interval, cancel ongoing request on external trigger
+          tap(([isSettled, connectionStatus]) => {
+            if (connectionStatus === ConnectionStatus.down) {
+              // REVIEW: slightly redundant since connection debug messages are already recorded separately
```

Comment:
When looking at the overall debug log, user will see a 'Connection down' message anyway.
Thus this one is redundant, but I think it's nice to see it here, in context too

Reply:
How about just logging `Skipping request` then?

Reply:
Yeah, I think we can keep it simple for now and assume it will look informative enough. We can always add more info later

Reply:
Done in [bd1f8c0](https://github.com/input-output-hk/cardano-js-sdk/pull/379/commits/bd1f8c0a0a499a2060572ee911828d2848edbb9d)